sycl/include/sycl/handler.hpp

```diff
@@ -2551,6 +2551,30 @@ class __SYCL_EXPORT handler {
     CGData.MAccStorage.push_back(std::move(AccImpl));
   }
 
+private:
```

Comment:
NIT: shall we put this to the main private section below where all helper methods are located and name like majority of the methods with lower case: stageFillCG?